src/cb_admin/pages/Activities.js

```diff
@@ -169,25 +171,35 @@ export default class ActivitiesPage extends React.Component {
           to your visitors, and just deselect all days if the one you are editing is a one-off
           event or not currently on the agenda.
         </Paragraph>
-        <Form onSubmit={this.addActivity} onChange={this.onChange}>
-          <LabelledInput
-            id="cb-admin-activities-name"
-            label="Add an activity"
-            name="name"
-            type="text"
-            error={errors.activity}
-            required
-          />
-          <LabelledSelect
-            id="cb-admin-activities-category"
-            label="Category"
-            name="category"
-            options={this.state.categories}
-            error={errors.activity}
-            required
-          />
-          <SubmitButton type="submit">ADD</SubmitButton>
-        </Form>
+        <Grid>
```

Comment:
Can these `Grid` changes be separated out into a separate PR? I don't want them to be squashed into this PR incase this feature is never implemented.

Reply:
From what I can see, the only changes that can be separated are those to `homeVisitor.js`. Just want to confirm that this is all you were expecting.

Reply:
@eliascodes were the other changes from the styling PR?